app/build.gradle

```diff
@@ -21,17 +22,28 @@ android {
     }
     buildFeatures {
         viewBinding = true
+        dataBinding true
     }
 }
 
 dependencies {
     def nav_version = "2.3.5"
+    def retrofit_version = "2.9.0"
+    def glide_version = "4.12.0"
 
     //Navigation
     implementation("androidx.navigation:navigation-fragment-ktx:$nav_version")
     implementation("androidx.navigation:navigation-ui-ktx:$nav_version")
 
+    //Retrofit
+    implementation "com.squareup.retrofit2:retrofit:$retrofit_version"
+    implementation 'com.squareup.retrofit2:converter-gson:2.5.0'
+
+    //Glide
+    implementation "com.github.bumptech.glide:glide:$glide_version"
+
     implementation 'androidx.core:core-ktx:1.7.0'
     implementation 'androidx.appcompat:appcompat:1.3.1'
     implementation 'com.google.android.material:material:1.4.0'
+    implementation "androidx.activity:activity-ktx:1.4.0"
```

Comment:
since you've started to extract versions as variables, let's make it for all dependencies

Reply:
done